rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/rules/CreateReportIndexRuleProvider.java

```diff
@@ -27,7 +27,7 @@
 @RuleMetadata(phase = ReportGenerationPhase.class)
 public class CreateReportIndexRuleProvider extends AbstractRuleProvider
 {
-    public static final String REPORT_INDEX = "Report Index";
+    public static final String REPORT_INDEX = "Dashboard";
```

Comment:
Yes, I meant the additional report links at the bottom of the page. We used to treat this page as a guide to the other reports rather than as a "dashboard", but I think that Marc views it differently.